docs/DEPLOYMENT.md

```diff
@@ -29,6 +29,16 @@
 
 Lets build and push all images to a registry and then create deployments.
 
+**Note** : You can use images from quay and skip the build image section
```

Comment:
image tag would be same as release tag. you can check the latest release [here](https://github.com/tektoncd/hub/releases)
for eg. if release tag is v1.3.0 same would be the image tag i.e. quay.io/tekton-hub/api:v1.3.0